index.js

```diff
@@ -8,10 +8,23 @@ var postcss = require("postcss")
 var mime = require("mime")
 var url = require("url")
 var SvgEncoder = require("directory-encoder/lib/svg-uri-encoder.js")
-var reduceFunctionCall = require("reduce-function-call")
 var mkdirp = require("mkdirp")
 var crypto = require("crypto")
 var pathIsAbsolute = require("path-is-absolute")
+/**
+ * @typedef UrlRegExp
+ * @name UrlRegExp
+ * @desc A regex for match url with parentheses: (before url)(the url)(after url).
+ * (the url) will be replace with new url, and (before) and (after) will remain
+ * @type RegExp
+ */
+/**
+ * @type {UrlRegExp[]}
+ */
+var UrlsPatterns = [
+  /(url\(\s*['"]?)([^"')]+)(["']?\s*\))/g,
+  /(AlphaImageLoader\(\s*src=['"]?)([^"')]+)(["'])/g,
```

Comment:
Now it is easy to add other url formats. Just add a regular expression, and it will be processed.